public/stylesheets/local.css

```diff
@@ -186,10 +186,12 @@ table td > .badge {
 }
 
 /* Used to give table rows the same padding as Bootstrap card headers */
-.card table tr td:first-child, .card table tr th:first-child {
+.card > table tr td:first-child, .card > table tr th:first-child,
+.card .table-responsive > table tr td:first-child, .card .table-responsive > table tr th:first-child {
```

Comment:
Do we want that to be `.card > .table-responsive`?

Reply:
I also think I prefer the 4-line version, fwiw.

Reply:
Sure, this seems like a safe change to make.

Reply:
Super minor nit, but the 4-line version was easier for me to see what was going on.

Reply:
hah, sure